pages/register/[domainToRegister].tsx

```diff
@@ -25,23 +25,20 @@ const RegistrationPage: NextPage = () => {
   }, [router]);
 
   return (
-    <div className={homeStyles.screen}>
-      <div className={styles.container}>
-        <div className="sm:w-2/5 w-4/5 mt-5 mb-5">
-          <SearchBar
-            onChangeTypedValue={(typeValue: string) => setDomain(typeValue)}
-            showHistory={false}
+    <FormProvider>
```

Comment:
It's a detail but it makes more sense to not wrap the two divs screen and container and just take the register/renewal component.